res2df/pillars.py

```diff
@@ -330,12 +332,12 @@ def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
     """
     parser.add_argument(
         "DATAFILE",
-        help=("Name of Eclipse DATA file. " "INIT and EGRID file must lie alongside."),
+        help=("Name of reservoir .DATA file. INIT and EGRID file must lie alongside."),
```

Comment:
parser.add_argument(
"DATAFILE", help="Name of the .DATA input file for the reservoir simulator. There must exist .INIT and .EGRID files with the same path and basename."